pygments/lexers/configs.py

```diff
@@ -1127,3 +1127,99 @@ def analyse_text(text):
             result += 0.49
 
         return result
+
+
+class PasswdLexer(RegexLexer):
+    """
+    Lexer for /etc/passwd files.
+    """
+
+    name = 'PasswdLexer'
+    aliases = ['passwd']
+    filenames = ['passwd']
```

Comment:
I find this a bit problematic. It will assume any file called "passwd" is in /etc/passwd format. I think "passwd" might be too generic a name to make that assumption. Also, is it really useful? I mean, I'd expect a program that uses Pygments to highlight a file without providing a language explicitly to pass a file that is part of some kind of project (LaTeX document, Git repository, Sphinx documentation or such), not part of the system itself. Could we leave `filenames` empty and let the user explicitly specify to use the `/etc/passwd` lexer on the command line or via the tool they are using?